src/Common/PODArray.h

```diff
@@ -92,11 +93,12 @@ class PODArrayBase : private boost::noncopyable, private TAllocator    /// empty
     /// Round padding up to an whole number of elements to simplify arithmetic.
     static constexpr size_t pad_right = integerRoundUp(pad_right_, ELEMENT_SIZE);
     /// pad_left is also rounded up to 16 bytes to maintain alignment of allocated memory.
-    static constexpr size_t pad_left = integerRoundUp(integerRoundUp(pad_left_, ELEMENT_SIZE), 16);
+    static constexpr size_t pad_left = integerRoundUp(pad_left_, std::lcm(ELEMENT_SIZE, 16));
```

Comment:
It's difficult to understand, for what values of ELEMENT_SIZE there is a difference?

Reply:
24, 40 and 88 from what I see in build failures without that fix.